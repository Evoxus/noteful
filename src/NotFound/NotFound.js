import React from 'react';

export default function NotFound(props) {
  return(
    <h2>The page you are looking for doesn't exist</h2>
  )
}