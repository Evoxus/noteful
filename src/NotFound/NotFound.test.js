import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import NotFound from './NotFound';

describe('App component', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<NotFound />, div);
    ReactDOM.unmountComponentAtNode(div);
  })
  it('renders empty given no tabs', () => {
    const wrapper = shallow(<NotFound />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})