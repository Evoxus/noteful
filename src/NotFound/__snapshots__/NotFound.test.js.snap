// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component snapshot test 1`] = `
<h2>
  The page you are looking for doesn't exist
</h2>
`;
